zerver/tornado/descriptors.py

```diff
@@ -16,3 +16,15 @@ def set_descriptor_by_handler_id(handler_id: int, client_descriptor: "ClientDesc
 
 def clear_descriptor_by_handler_id(handler_id: int) -> None:
     del descriptors_by_handler_id[handler_id]
+
+
+current_port: Optional[int] = None
+
+
+def is_current_port(port: int) -> Optional[int]:
+    return current_port is None or current_port == port
```

Comment:
What's the purpose of the `current_port is None` case; is that for `!RUNNING_INSIDE_TORNADO` aka the test suite, or something else?

Reply:
Yeah, for the test suite. (Maybe `not settings.RUNNING_INSIDE_TORNADO` or `settings.TEST_SUITE` would be clearer?)

Reply:
Yeah, `settings.TEST_SUITE` would be clearer; we could add a comment instead if desired.